src/app/groups/containers/user/user.component.html

```diff
@@ -1,11 +1,12 @@
 <ng-container *ngIf="state$ | async as state">
 
-  <alg-loading *ngIf="state.isFetching"></alg-loading>
+  <alg-loading class="alg-flex-1" *ngIf="state.isFetching"></alg-loading>
```

Comment:
This does not seem to work, the page is blank without loader while it is waiting. 

https://github.com/France-ioi/AlgoreaFrontend/assets/1053150/08df321c-a97a-4d7b-b4b6-02f46e2a428e

Reply:
Fixed